cmake/config/pkgconfig/deal.II.pc.in

```diff
@@ -0,0 +1,11 @@
+DEAL_II_PATH=@DEAL_II_PATH_PC@
+DEAL_II_INCLUDE_RELDIR=@DEAL_II_INCLUDE_RELDIR@
+libdir=${DEAL_II_PATH}/@DEAL_II_LIBRARY_RELDIR@
+includedir=@CONFIG_INCLUDE_DIRS@
```

Comment:
We will have to fix the path handling - I'll comment on this later today (when I have time to think about how to make this more flexible).